### PR TITLE
PrintService: Report startDateTime/endDateTime

### PR DESCRIPTION
**Pull request by kanchm**
Fixing documentation to use the parameters as specified in the schema.

### CODE REVIEW

File: api-reference/beta/api/reports-getgrouparchivedprintjobs.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The following is an example of the request.
   "name": "reports-getgrouparchivedprintjobs"
 }-->
 ```http
-GET https://graph.microsoft.com/beta/print/reports/getGroupArchivedPrintJobs(groupId='{id}',periodStart=<timestamp>,periodEnd=<timestamp>)
+GET https://graph.microsoft.com/beta/print/reports/getGroupArchivedPrintJobs(groupId='{id}',startDateTime='{timestamp}',endDateTime='{timestamp}')
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/reports-getgrouparchivedprintjobs.md:60]: GLOBAL:
The docs use the curly brackets around placeholders, not angle brackets.
I think there should be single quotes around each of the parameter values too.

File: api-reference/beta/api/reports-getgrouparchivedprintjobs.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ In addition to the following permissions, the user's tenant must have an active
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
 ```http
-GET /print/reports/getGroupArchivedPrintJobs
-GET /reports/getGroupArchivedPrintJobs
+GET /print/reports/getGroupArchivedPrintJobs(groupId=groupId-value,startDateTime=startDateTime-value,endDateTime=endDateTime-value)
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/reports-getgrouparchivedprintjobs.md:32]: GLOBAL:
The syntax section should also include the parameters.
The formatting here is used in other function topics with parameters, for consistency.

**Review by angelgolfer-ms — APPROVED** [review]
@kanchm , I made a couple other changes in each of these topics - syntax section to include parameters, and example request to use consistent formatting for placeholders around single quotes. Please see if they look right to you. If you think we should have the service owner confirm, please have the name specified in the author: field at the top of the corresponding topic review.

File: api-reference/beta/api/reports-getgrouparchivedprintjobs.md (hunk 1 above)
Comment (kanchm) [reply api-reference/beta/api/reports-getgrouparchivedprintjobs.md:60]: @angelgolfer-ms quotes are not allowed for datetime fields.

Tried this in Graph Explorer.
https://graph.microsoft.com/beta/print/reports/getGroupArchivedPrintJobs(groupId='{id}',startDateTime=2020-01-01,endDateTime=2020-01-02)?$whatif

If I add quotes, then OData parser gives this error:
`"Expression of type 'Edm.String' cannot be converted to type 'Edm.DateTimeOffset'."`

File: api-reference/beta/api/reports-getgrouparchivedprintjobs.md (hunk 2 above)
Comment (kanchm) [reply api-reference/beta/api/reports-getgrouparchivedprintjobs.md:32]: Sounds good. Should the groupId parameter be formatted with single quotes?

File: api-reference/beta/api/reports-getprinterarchivedprintjobs.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ The following is an example of the request.
   "name": "reports-getprinterarchivedprintjobs"
 }-->
 ```http
-GET https://graph.microsoft.com/beta/print/reports/getPrinterArchivedPrintJobs(printerId='{id}',periodStart=<timestamp>,periodEnd=<timestamp>)
+GET https://graph.microsoft.com/beta/print/reports/getPrinterArchivedPrintJobs(printerId='{id}',startDateTime='{timestamp}',endDateTime='{timestamp}')
```
Comment (kanchm) [inline api-reference/beta/api/reports-getprinterarchivedprintjobs.md:60]: Quotes need to be removed from datetime parameters.

File: api-reference/beta/api/reports-getuserarchivedprintjobs.md [hunk 4]
```diff
@@ -57,7 +57,7 @@ The following is an example of the request.
   "name": "reports-getuserarchivedprintjobs"
 }-->
 ```http
-GET https://graph.microsoft.com/beta/print/reports/getUserArchivedPrintJobs(userId={id},periodStart=<timestamp>,periodEnd=<timestamp>)
+GET https://graph.microsoft.com/beta/print/reports/getUserArchivedPrintJobs(userId='{id}',startDateTime='{timestamp}',endDateTime='{timestamp}')
```
Comment (kanchm) [inline api-reference/beta/api/reports-getuserarchivedprintjobs.md:60]: Quotes again.

File: api-reference/beta/api/reports-getgrouparchivedprintjobs.md (hunk 2 above)
Comment (angelgolfer-ms) [reply api-reference/beta/api/reports-getgrouparchivedprintjobs.md:32]: I don't see that in other existing function topics with parameters that were generated by the Ruby tool.

**Pull request merged**